### PR TITLE
Admin module

### PR DESCRIPTION
**Pull request by bnmajor**
Can currently view and remove groups, and view, add, or remove group members.

### CODE REVIEW

File: packages/girder-ui/src/admin/components/members/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+class Members extends Component {
+	render() {
+		const {group, listOfMembers, search} = this.props;
+		return(
+			<div style={{margin:'10px'}}>
+				<Typography style={{textAlign:'center'}} variant='h4' gutterBottom>
```
Comment (cjh1) [inline packages/girder-ui/src/admin/components/members/index.js]: Looks like we have some generous indenting here, 2 spaces should be enough.

File: packages/girder-ui/src/admin/components/users/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+							<TableCell>Name</TableCell>
+							<TableCell>Username</TableCell>
+							<TableCell>Add User</TableCell>
+						</TableRow>
+					</TableHead>
+					<TableBody>
+						{possibleUsers.map((person) => (
+							<TableRow key={person._id}>
+								<TableCell>
+									{person.firstName+' '+person.lastName}
```
Comment (cjh1) [inline packages/girder-ui/src/admin/components/users/index.js]: same here

File: packages/girder-ui/src/admin/components/members/index.js (hunk 1 above)
Comment (bnmajor) [reply packages/girder-ui/src/admin/components/members/index.js]: Corrected to be 2 spaces

File: packages/girder-ui/src/admin/components/users/index.js (hunk 2 above)
Comment (bnmajor) [reply packages/girder-ui/src/admin/components/users/index.js]: Fixed

**Pull request merged**